### PR TITLE
new/si-sw/blocked-view

### PR DESCRIPTION
**Pull request by SamWhite745**

### CODE REVIEW

File: app.rb
Comment (SamWhite745) [inline app.rb:1]: created helpers and the blocked paths

File: lib/user_repository.rb
Comment (SamWhite745) [inline lib/user_repository.rb:1]: Changed the functionality of find handle to return a handle instead of a user

File: spec/user_repository_spec.rb
Comment (SamWhite745) [inline spec/user_repository_spec.rb:1]: fixed the test to reflect the functionality change of find_handle

**Review by SamWhite745 — COMMENTED** [review]
I left some (hopefully) handy comments to explain the changes. Also, I finally removed the pesky coverage files - rejoice!

**Review by pablisch — APPROVED** [review]
I'm only mildly uncomfortable with the implicit redirect 😉

**Pull request merged**